Cleanup PWT

Dreeam-qwq opened the pull request:
This PR intends to fix very basically problems in PWT patch, code style, configurable. etc.

Makes PWT fully configurable. The "configurable" means that if PWT is disabled, all logic should behave as if the PWT patch does not exist, should not be affected by things in PWT.
Also to extract some changes to separate methods, to make the diff looks better and easier to track changes during the update.

Dreeam-qwq committed:
IPeiqi - Power Down And Start Again

Dreeam-qwq committed:
Anyma - A True Connection

Godl1nk commented:
No bad intentions but, its the third time i had to redo my ‘Remove leaf pwt patch’ on my fork 😭😭 Don’t redo pls

Dreeam-qwq committed:
BT - Skylarking (Original Mix)

Dreeam-qwq committed:
Coeus - Eden

Dreeam-qwq committed:
Remove one extra execute mid-tick task call in level tick when PWT is disabled

This may cause extremely rare, weird, strange, magic, mysterious issues with plugins, or potentially more.

One example is that it may cause boss mob duplication issue when `ONE MOB ONLY` was enabled in plugin SupremeBosses

Dreeam-qwq committed:
Gregory Esayan - Impressionism

Dreeam-qwq committed:
jeonghyoen - Say to Me

Dreeam-qwq committed:
HOMI/B4N4N4 G4NG - In Your Love

Dreeam-qwq committed:
Rameses B - To The Stars
